README.md

```diff
@@ -64,6 +64,54 @@ install docker and make it accessible on your PATH.
 
 The template "type" for this handler is `cdk`.
 
+### Python Stack Classes vs. cdk.json
+
+The template `path` argument can either direct to a Python file with a `SceptreCdkStack class` (see
+"Making your Python stack class" below) or it can point a `cdk.json` file. Template synthesis and
```

Comment:
```suggestion
The template `path` argument can either specify a Python file with a `SceptreCdkStack class` (see
"Making your Python stack class" below) or it can specify a `cdk.json` file. Template synthesis and
```